site/src/sections/Patterns/index.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PatternsStyleWrapper from "./patterns.style";
+import Container from "../../components/Layout/Container";
+import howItWorksVideo from "../../assets/videos/how-it-works.mp4";
```

Comment:
we should use webm format for videos instead of other formats. https://zeo.org/resources/blog/what-are-webp-and-webm-and-how-to-use-them

Reply:
Here we need to update the video, this video is added only for demo purpose. So when we update it we can use **webm** format definitely.

Reply:
@Udit-takkar Update video with webm format.